### PR TITLE
Fixing #10185: modified srepr printing docs.

### PR DESCRIPTION
**Pull request by Shekharrajak**
Added some Usage/examples for srepr and showed how repr different from srepr printing.

### CODE REVIEW

**Comment by Shekharrajak** [comment]
Fixing [10185](https://github.com/sympy/sympy/issues/10185)

File: doc/src/modules/printing.rst [hunk 1]
```diff
@@ -307,11 +307,27 @@ This class implements Python printing. Usage::
...
 -----------
 
 .. module:: sympy.printing.repr
 
+SymPy doesn’t use repr() for generating textual representation of expressions.
+
+Usage ::
+
+    >>> repr(5*x**3 + sin(x))
+    5*x**3 + sin(x)
```
Comment (gxyd) [inline doc/src/modules/printing.rst]: This should have quotes around it `'5*x**3 + sin(x)'`

Comment (Shekharrajak) [inline doc/src/modules/printing.rst]: Done. thanks.

**Comment by Shekharrajak** [comment]
@asmeurer please help to pass all cases,if possible.

**Comment by leosartaj** [comment]
You need to fix the merge conflicts.

**Comment by Shekharrajak** [comment]
Please give me any hint (comment in line of code) if you find something missing or wrong syntax.
thanks @leosartaj

**Comment by asmeurer** [comment]
I didn't see this. Apparently there is duplication here between https://github.com/sympy/sympy/pull/10253 and https://github.com/sympy/sympy/pull/10225 (the second of which was merged, hence the merge conflicts here).

**Comment by asmeurer** [comment]
@sahilshekhawat, @sampadsaha5 how would you like to proceed here?

**Comment by gxyd** [comment]
@Shekharrajak you need to first fix the merge conflicts only then others can point you to mistakes easily. You can [read this](https://help.github.com/articles/resolving-a-merge-conflict-from-the-command-line/) if you want to know how to fix the conflicts.

**Comment by Shekharrajak** [comment]
I tried to see the conflict using `git status` but it is saying `working directory clean`
Anyways I tried to make bigger change in commit to resolve conflict ,but still it has conflicts.

**Comment by asmeurer** [comment]
@Shekharrajak in the future, please use a different branch from master to make your changes (I believe the contributing guide explains how to do this). Normally to resolve conflicts you would pull master and merge it into your branch, but since you committed on master, likely just `git pull` will do what you need. That will tell you that some files have conflicts. You should then go through them and fix them all.

**Comment by Shekharrajak** [comment]
I saw some PR where they want to merge from  their master branch , so I followed their approach.After this I am using different branches for different issues on which I am working.I try to fix this.

**Comment by Shekharrajak** [comment]
I reverted the file changes.

**Pull request closed without merging**